### DISCUSSION TITLE
Help: FEC patch and map update on Audi A6 c7 encounter problems

### DISCUSSION THREAD

**Discussion by Csall67** [Mr-MIBonk/M.I.B._More-Incredible-Bash#101] [Q&A] [answered]
Hello, I operate according to the program. My firmware is MHI2_ CN_ AU57x_ K1006_ 2. The downloaded patch is MHI2_ CN_ AU57x_ K1006_ MU1005_ PATCH, the patch cannot be found by direct installation. I modified the folder name MHI2_ CN_ AU57x_ K1006_ 2MU1005_ PATCH, installation completed.

But when I checked fec, I added 02320044, which is legal. But 00030000, 00040100, 00050000, 00060B00, 00060800, 00060900, 00070200 become permanently blocked. How can I solve it?thank you!

**Comment by harman-f** [comment, marked as answer]
Well, quite a bit of forth and back in the LOG.
You patched, copies stock ifs-root back and patched again...

Run: M.I.B. -> Advanced Settings -> SWDL CLEANER

**Comment by Csall67** [comment]


**Reply by harman-f** [reply]
Run in M.I.B GEM:

Advanced Settings -> SWDL CLEANER
Reboot
Check FECs again in menu

**Reply by Csall67** [reply]
I ran it. There is only 02320044 in fec.Nothing else.
Do you still need to send the log file to you?

**Reply by harman-f** [reply]
please share a zip of your backup folder from SD card

**Reply by harman-f** [reply]
![grafik](https://user-images.githubusercontent.com/98130152/216734845-ef203abc-bf52-4e30-b23f-66c4d230c5c3.png)

No Idea right away what is going on on your unit.
Original FecContainer in backup looks OK.

Use restore function in GEM to restore your FECContainer.

**Comment by harman-f** [comment]
Just run 3 and reboot

**Reply by harman-f** [reply]
Restore FecContainer …

**Reply by Csall67** [reply]
I modify the folder name MHI2_CN_AU57x_K1006_MU1005_PATCH，For firmware MHI2_CN_AU57x_K1006_2_MU1005，Is wrong。
Is that the problem?

**Reply by harman-f** [reply]
No, patch is the same for both version. 

Run addFec function in GEM

**Reply by harman-f** [reply]
You have to run addFec again now.

**Reply by Csall67** [reply]
I have run addFec, added 023200ee.Thank you again!

**Reply by MEGI313** [reply]
Hello, I have a problem with updating maps after adding the fec code 023000EE, the map update option is not available, does anyone have any ideas?
